patches/rrweb@2.0.0-alpha.17.patch

```diff
@@ -0,0 +1,482 @@
+diff --git a/dist/rrweb.js b/dist/rrweb.js
+index 12f4ad3c7f897fe8a9ee2f4595dcf37f302d17c5..65b80aa52851aa8d57bfa218e639a850f1b19cb7 100644
+--- a/dist/rrweb.js
++++ b/dist/rrweb.js
+@@ -15007,7 +15007,7 @@ class MediaManager {
+     }
+   }
+   addMediaElements(node2, timeOffset, mirror2) {
+-    if (!["AUDIO", "VIDEO"].includes(node2.nodeName)) return;
++    if (!this.isSupportedMediaElement(node2)) return;
+     const target = node2;
+     const serializedNode = mirror2.getMeta(target);
+     if (!serializedNode || !("attributes" in serializedNode)) return;
+@@ -15060,6 +15060,7 @@ class MediaManager {
+     timeOffset,
+     mutation
+   }) {
++    if (!this.isSupportedMediaElement(target)) return;
+     this.mediaMap.set(
+       target,
+       this.getMediaStateFromMutation({
+@@ -15312,7 +15313,11 @@ class Replayer {
+       mouseTail: defaultMouseTailConfig,
+       useVirtualDom: true,
+       // Virtual-dom optimization is enabled by default.
+-      logger: console
++      logger: console,
++      onError: (e) => {
++        // maintain the existing behaviour of throwing if no handler is provided
++        throw e;
++      },
```

Comment:
Originally introduced in https://github.com/PostHog/posthog/pull/19868. Do you want to keep this around @pauldambra? If we removed it we'd be down to a single one line patch (which has an open PR from Justin) on the playback side

Reply:
interesting... doesn't look like we're setting a handler
so this feels orphaned to me
if you concur there is no handler than i humbly submit it can be terminated with medium prejudice

Reply:
Handler is here: https://github.com/PostHog/posthog/blob/859f7221666413ac8c543ee0ed9fa2bea244ae4a/frontend/src/scenes/session-recordings/player/sessionRecordingPlayerLogic.ts#L625-L627

Reply:
ah, intereseting, caught 110 errors in the last 30 days https://posthog.sentry.io/issues/?project=1899813&query=is%3Aunresolved%20issue.priority%3A%5Bhigh%2C%20medium%5D%20feature%3A%22replayer%20error%20swallowed%22&referrer=issue-list&statsPeriod=30d&utc=true

(Which we haven't looked at 🙈 )